kwarchart/src/main/java/com/kwarchart/android/Chart.kt

```diff
@@ -273,12 +273,12 @@ fun DrawScope.drawGrids(
     count: Int,
     color: Color,
     xAxisEndPadding: Float = 0f,
+    yAxisEndPadding: Float = 0f,
     showHorizontalLines: Boolean = true,
     showVerticalLines: Boolean = true,
 ) {
-    val hGap = size.height / count
+    val hGap = (size.height - yAxisEndPadding) / count
     val vGap = (size.width - xAxisEndPadding) / count
-
     val startPoint = origin()
 //    val pathEffect = PathEffect.dashPathEffect(intervals = floatArrayOf(10f, 10f))
```

Comment:
pls remove if not used

Reply:
Will be used in the future. 
v0.0.2: https://docs.google.com/spreadsheets/d/1dGGRSZ9Z4gJ-M44rUmFNp8CXjK8dtgqxttfLj8FG69M/edit#gid=0